content/es/docs/concepts/overview/components.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- raelga
+title: Componentes de Kubernetes
+content_type: concept
+weight: 20
+card:
+  name: concepts
+  weight: 20
+---
+
+<!-- overview -->
+
+Este documento describe los distintos componentes que
+son necesarios para operar un clúster de Kubernetes.
+
+<!-- body -->
+
+## Componentes maestros
+
+Los componentes maestros proveen el plano de control del clúster. Ellos toman decisiones globales sobre
+el clúster (por ejemplo, la planificación) y detectan y responden a eventos del clúster, como la creación
+de un nuevo pod cuando la propiedad `replicas` de un controlador de replicación no se cumpla.
+
+Estos componentes pueden ejecutarse en cualquier nodo del clúster. Sin embargo, y para simplificar, los
+scripts de instalación típicamente inician todos los componentes maestros en el mismo nodo de forma exclusiva,
+sin ejecutar contenedores de los usuarios en esos nodos. Vea [Construyendo clústeres de alta disponibilidad](/docs/admin/high-availability/) para un ejemplo de configuración multi-master.
```

Comment:
WG Naming is replacing the term “master” with “control plane”. Please consider whether another translation is more appropriate.

Reply:
`master` wording removed in https://github.com/kubernetes/website/pull/22259/commits/40ca6e686e6631cd400b0db11701afa8e3f31091